insecure/integration/test/composability_test.go

```diff
@@ -48,6 +48,11 @@ func TestCorruptibleConduitFrameworkHappyPath(t *testing.T) {
 				// implementing the corruption functionality of the orchestrator.
```

Comment:
I feel like we're using the word "attacker" and "orchestrator" interchangeably. We should probably stick with one name to be less confusing

Reply:
In this file we are not using the _attacker_ and _orchestrator_ interchangeably. Basically, we use the term an _attacker_ to refer to the combination of an _orchestrator_ that is running on top of an _attack network_. The term attacker provides an abstraction that encapsulates the attacker-side behavior without relying on the implementation details such as decoupling the attacker logic further into the attack network and orchestrator. I am going to clarify this point in the readme of the `corruptible` package (in a separate PR).